app/views/outcomes-report/OutcomesReportResultComponent.vue

```diff
@@ -92,48 +107,106 @@ export default Vue.extend({
     },
 
     licenses () {
+      // licenses are in shared db so do not merge coco & oz
       return this.org.newLicenses || []
     },
 
     totalLicense () {
       return this.licenses.reduce((acc, license) => {
         acc.used += license.used
         acc.count += license.count
-        license.teachers.forEach(tea => acc.teachers.add(tea))
-        license.schools.forEach(sch => acc.schools.add(sch))
+        license.teachers?.forEach(tea => acc.teachers.add(tea))
+        license.schools?.forEach(sch => acc.schools.add(sch))
         return acc
       }, { used: 0, count: 0, teachers: new Set(), schools: new Set() })
     },
 
     coursesWithProgress () {
-      if (!this.org.progress && !this.org.newReport) return []
+      if (!this.org.progress) return []
       let courses = _.cloneDeep(this.sortedCourses)
-      let alreadyCoveredConcepts = []
-      for (const course of courses) {
-        course.name = utils.i18n(course, 'name')
-        course.acronym = utils.courseAcronyms[course._id]
-        course.studentsStarting = (this.org.progress.studentsStartingCourse || {})[course._id] || 0
-        course.studentsCompleting = (this.org.progress.studentsCompletingCourse || {})[course._id] || 0
-        course.completion = course.studentsStarting ? Math.min(1, course.studentsCompleting / course.studentsStarting) : 0
-        if (this.org.newReport) {
-          const courseCompleteLevels = this.org.progress.courseCompleteLevels || {}
-          const courseStartingStudents = this.org.progress.courseStartingStudents || {}
-          const completeLevels = Math.max(...Object.values(courseCompleteLevels[course._id] || {})) || 0
-          course.studentsStarting = courseStartingStudents[course._id] || 0
-          course.completeLevels = completeLevels
-          course.completion = Math.min(1, course.completeLevels / (this.org.progress.courseAllLevels[course._id] || 1))
-        }
-        course.newConcepts = _.difference(course.concepts, alreadyCoveredConcepts)
-        alreadyCoveredConcepts = _.union(course.concepts, alreadyCoveredConcepts)
-      }
-      if (this.org.newReport) {
-        courses = _.filter(courses, (course) => course.completeLevels >= 1)
-      } else {
-        courses = _.filter(courses, (course) => (course.studentsStarting + course.studentsCompleting) >= Math.min(100, Math.max(2, Math.ceil(0.02 * this.org.progress.studentsWithCode))))
+      courses = this.formatCourse(courses, this.org.progress, this.org.newReport)
+      if (this.showOther && this.otherOrg.progress) {
+        let otherCourses = _.cloneDeep(this.sortedOtherCourses)
+        otherCourses = this.formatCourse(otherCourses, this.otherOrg.progress, this.otherOrg.newReport)
+        courses = courses.concat(otherCourses)
       }
+
       return courses
     },
 
+    schoolCount () {
+      let schools = this.org.progress.schools
```

Comment:
use `schoolCount` - there is no schools key

Reply:
oh oh thank you. copilot did it, i should check